docs/exclusive-content.md

```diff
@@ -212,3 +204,41 @@ If you see the source files instead of the example, click **View App** in the bo
+
+[![](https://mermaid.ink/img/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5Vc2VyJ3MgQ2xpZW50IC0-PiBDcmVhdG9yJ3MgV2FsbGV0OiBzdHJlYW1zIHBheW1lbnRzXG5DcmVhdG9yJ3MgV2FsbGV0IC0-PiBVc2VyJ3MgQ2xpZW50OiBTVFJFQU0gcmVjZWlwdHNcblVzZXIncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogQ3JlYXRvcidzIHBheW1lbnQgcG9pbnRlciwgZW5jcnlwdGVkIHZlcmlmaWVyIGVuZHBvaW50LCBTVFJFQU0gcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBFeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3I6IGRlcml2ZXMga2V5LCBkZWNyeXB0cyB2ZXJpZmllciBlbmRwb2ludCwgdmVyaWZpZXMgcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBVc2VyJ3MgQ2xpZW50OiBrZXlcblVzZXIncyBDbGllbnQgLT4-IFVzZXIncyBDbGllbnQ6IGRlY3J5cHRzIGNvbnRlbnQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)](https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG5hdXRvbnVtYmVyXG5Vc2VyJ3MgQ2xpZW50IC0-PiBDcmVhdG9yJ3MgV2FsbGV0OiBzdHJlYW1zIHBheW1lbnRzXG5DcmVhdG9yJ3MgV2FsbGV0IC0-PiBVc2VyJ3MgQ2xpZW50OiBTVFJFQU0gcmVjZWlwdHNcblVzZXIncyBDbGllbnQgLT4-IEV4Y2x1c2l2ZSBDb250ZW50IEdlbmVyYXRvcjogQ3JlYXRvcidzIHBheW1lbnQgcG9pbnRlciwgZW5jcnlwdGVkIHZlcmlmaWVyIGVuZHBvaW50LCBTVFJFQU0gcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBFeGNsdXNpdmUgQ29udGVudCBHZW5lcmF0b3I6IGRlcml2ZXMga2V5LCBkZWNyeXB0cyB2ZXJpZmllciBlbmRwb2ludCwgdmVyaWZpZXMgcmVjZWlwdFxuRXhjbHVzaXZlIENvbnRlbnQgR2VuZXJhdG9yIC0-PiBVc2VyJ3MgQ2xpZW50OiBrZXlcblVzZXIncyBDbGllbnQgLT4-IFVzZXIncyBDbGllbnQ6IGRlY3J5cHRzIGNvbnRlbnQiLCJtZXJtYWlkIjp7InRoZW1lIjoiZGVmYXVsdCJ9LCJ1cGRhdGVFZGl0b3IiOmZhbHNlfQ)
+
+### Interactive example
+
+This example page will request the decryption key for each piece of exclusive content if Web Monetization is enabled. If it is not enabled, it will say "🔒 This content is exclusive for users with Web Monetization enabled."
+
+The example does require you to have Web Monetization enabled in your browser, hence you need to [open the page](https://exclusive-content-demo.glitch.me/) with Web Monetization enabled.
+
+If you see the source files instead of the example, click **View App** in the bottom right.
+
+<div class="glitch-embed-wrap" style={{ height: '420px', width: '100%' }}>
+  <iframe
+    src="https://glitch.com/embed/#!/embed/exclusive-content-demo?path=index.html&previewSize=100"
+    title="exclusive-content-demo on Glitch"
+    allow="geolocation; microphone; camera; midi; vr; encrypted-media"
```

Comment:
you need to add "monetization" to the allow directive

Reply:
I added it but I still see 
```
<iframe> (or one of its ancestors) is not authorized to allow web monetization, https://exclusive-content-demo.glitch.me/
```

Reply:
I see, did you check all the iframes in devtools? 
The whole chain has to have allow="monetization" 
If it does, then it could possibly be a bug in the policy parser

Reply:
For record, we figured out it was: 
https://support.glitch.com/t/feature-request-provide-a-way-to-customize-glitch-embeds-allow-attribute/31488

Reply:
@sublimator and I figured out that this is due to the way glitch embeds work. I'll create a preview demo that does not depend on wm to be enabled, just like the other examples on that page.